### PR TITLE
Fix serverless deployment

### PR DESCRIPTION
**Pull request by inigomarquinez**
### Main changes

- fix: add AWS_ACCOUNT_ID to serverless environment

### CODE REVIEW

**Review by w3dani — CHANGES_REQUESTED** [review]
Have a look to this -> https://www.serverless.com/framework/docs/providers/aws/guide/variables#referencing-aws-specific-variables

**Comment by inigomarquinez** [comment]
> Have a look to this -> https://www.serverless.com/framework/docs/providers/aws/guide/variables#referencing-aws-specific-variables

@w3dani , let me know if it's ok now

File: serverless.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ provider:
         - 'secretsmanager:GetSecretValue'
       Resource: 'arn:aws:secretsmanager:${self:provider.region}:${self:provider.environment.AWS_ACCOUNT_ID}:*'
```
Comment (w3dani) [inline serverless.yml:18]: here you can replace those values as well

**Review by w3dani — CHANGES_REQUESTED** [review]

File: serverless.yml [hunk 2]
```diff
@@ -7,15 +7,15 @@ useDotenv: true
...
+  region: ${aws:region}
   stage: ${env:ENVIRONMENT}
   timeout: 180 # 3 minutes
   memorySize: 128 # MB
   iamRoleStatements:
     - Effect: 'Allow'
       Action:
         - 'secretsmanager:GetSecretValue'
-      Resource: 'arn:aws:secretsmanager:${self:provider.region}:${self:provider.environment.AWS_ACCOUNT_ID}:*'
+      Resource: 'arn:aws:secretsmanager:${self:provider.region}:${aws:accountId}:*'
```
Comment (w3dani) [inline serverless.yml:18]: `${self:provider.region}` -> `${aws:region}`

**Review by w3dani — CHANGES_REQUESTED** [review]

File: serverless.yml (hunk 2 above)
Comment (inigomarquinez) [reply serverless.yml]: Done

**Review by w3dani — APPROVED** [review]

**Pull request merged**